### ISSUE TITLE
Design: Increase legibility of infobox text for "Your project level is 0!" page

### ISSUE THREAD

**Issue by thetanmancan** [hackforla/tdm-calculator#1935]
### Overview
The text of the infobox on the "Your project level is 0!" page is hard to read. Redesign the text and background so that the infobox is more legible.

### Details
<details><summary>Current infobox text</summary>

![image](https://github.com/user-attachments/assets/223a289b-c6ab-4308-8f4c-9227422642e8)
![Image](https://github.com/user-attachments/assets/06ec50d8-f055-45f5-9bbb-c94a42f70a10)
</details> 

There are two obvious legibility issues.
1. There is too little contrast between the text and the background color. ([Fails WCAG contrast requirements](https://webaim.org/resources/contrastchecker/?fcolor=C3391D&bcolor=F9DFDA))
2. The text is too small.

These issues should be resolved while maintaining consistency with the style guide. If there is time, other opportunities to increase legibility should be researched and implemented.

### Action Items
- [x] Research legibility guidelines.
- [x] Create a Figma prototype that reconciles proposed legibility solutions with the style guide.
- [x] Present prototype to Design team and iterate based on feedback if needed.
- [x] Review with Product, Dev and Research and iterate based on feedback if needed.
- [x] Once finalized, add Before and After images to the [staging deck on this slide](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit#slide=id.g30e773e7a7d_1_87).
- [x] Get Stakeholder sign-off.
- [x] Write developer handoff issue.
- [x] Update the [Style Guide Presentation](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit#slide=id.g2f74a24ec8f_0_8) with the changes from this issue.
- [x] Update the [Figma design system](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=3331-11987&t=TrlU0u2UcvalSyGh-1).

### Resources
- [Link to TDM Calculator Dev site](https://tdm-dev.azurewebsites.net/calculation/1/0)
- Steps to replicate the warning infobox on the website
  - Navigate to TDM Calculator "Create Project" page.
  - Fill in the required fields on page 1 (Project Name, Address, AIN/APN).
  - Navigate to the next page using the navigation button on the bottom.
  - Without filling any details on page 2, navigate to the next page using the navigation buttons.
  - You'll see the "Project level 0" warning on the page.
- [Link to style guide presentation](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit#slide=id.g2a6d1bfeb46_0_103)
- [Figma page with the current designs for infoboxes](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=14107-737&t=zvmqcp8DxikCNpdt-1)
- [Visual design rules that can help with increasing legibility](https://anthonyhobday.com/sideprojects/saferules/)

### Development Issue
- #2059

**Comment by mjldesign** [comment]
When navigating to the indicated page, I am not seeing the issue of contrast/legibility. Here is what is currently showing on the dev site: 

![Image](https://github.com/user-attachments/assets/d9eb1ce5-b13d-48b9-9885-1c39eded5cc0)

* Can you clarify what needs to be changed, specifically when it comes to contrast/legibility? 
* Since this issue was created over a month ago, has this already been addressed or fixed?

**Comment by thetanmancan** [comment]
@mjldesign Admittedly, I've been out of the loop for the last two weeks, so I'm not caught up on what's been going on. That being said, I do recall during one of our meetings after I was asked to write this issue that Bonnie asked John to make some temporary changes on the dev site related to this issue. I think it was because the production site and the dev site had the same content and they didn't want the ugly, unreadable version of this infobox being seen or used by clients.

tl;dr What you're seeing on the dev site is a temporary solution and we still need a final design for the developers to implement.

Feel free to correct me if I'm wrong about any of this, @NilakshiS

**Comment by mjldesign** [comment]
Created two options to solve legibility for this page. 

Option 1:
* Header 4 text style
* Error text text color
* Error background rectangle behind text

![Image](https://github.com/user-attachments/assets/1953a07f-d46a-41e5-a4c4-46bd5e0c14a3)

Option 2:
* Header 4 text style
* Black text color
* Error background rectangle behind text

![Image](https://github.com/user-attachments/assets/48e7d1c5-ba1e-48aa-84d5-6767e47dcd77)

**Comment by NilakshiS** [comment]
From team meeting on 2024-12-11:
- Option 1 is not viable, because it still fails the contrast requirements
- Option 2 was discussed, team suggested some improvements such as changing the image to a warning icon. Launch icon for external links needs to be added to the link.

**Comment by thetanmancan** [comment]
Hi @mjldesign. Just wanted to check in and see if you've had any updates on this.

**Comment by thetanmancan** [comment]
Hey @mjldesign. I hope you had a nice winter vacation. Since we're back in action as of this week, please provide an update for this issue using the following format by Tuesday night. Thank you!

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by mjldesign** [comment]
1. Progress: 
- Updated designs based on previous meeting feedback. 
- Replaced image with warning icon (both triangle and circle). 
- Proposed two options for header color.
- Tidied up text layout and spacing. 
- Awaiting feedback to proceed with any further changes.

2. Blockers:
- No difficulties or errors currently encountered.

3. Availability:
- I have the availability to implement any changes/updates based on feedback within 3 days.

4. ETA: 
- Issue can be completed once designs have been approved. I don't foresee any additional time necessary.

5. Current Updates
- Black header, both circle and triangle warning icon options:
![Black Header](https://github.com/user-attachments/assets/a21e2645-5ff6-458b-85f8-192ea6950aec)
- Red header, both circle and triangle warning icon options:
![Red Header](https://github.com/user-attachments/assets/17d201d1-1845-4fac-8a8c-c3ffcfa9770c)

**Comment by thetanmancan** [comment]
@mjldesign Here is some feedback from the design team's 2025-1-08 meeting.

- The design with the triangle warning icon got the most votes because it conveys the importance of what is being warned and is consistent with the warnings we have in our system.
- The design with black header was chosen because it's consistent with the design used in modals and is harder to read when it's in red.
- I noticed in the Figma files that the font being used isn't Calibri. Please make sure your final design uses Calibri in all the fonts.
- Please remove the "Page 3/5" text and right arrow button at the bottom because the user no longer has the ability to move forward in the application process once they navigate this page.
- Please remove the exclamation point in the title since punctuation is not used in page titles in our system with the exception of question marks.
- The red background color of the text still makes it a bit hard to read. For another issue, #1916 , Grace had proposed using a lighter shade of red for the warning tooltip she was designing, #FFEDEA. I'm going to run that shade of red by the team leads at Wednesday's meeting. If they approve it, we will use that as the new red background color in our design system that you can apply for this page.

I think that's it. If you have any other changes you'd like to propose, please let me know. Thank you.

**Comment by mjldesign** [comment]
**1. Progress:**

* Updated designs based on previous meeting feedback.
* Changed font to Calibri
* Changed red background box to #FFEDEA
* Removed right nav arrow + "Page 3/5"; added "Start Over" button
* Awaiting feedback to proceed with any further changes.

**2. Blockers:**

No difficulties or errors currently encountered.

**3. Availability:**

I have the availability to implement any changes/updates based on feedback within 3 days.

**4. ETA:**

Issue can be completed once designs have been approved. I don't foresee any additional time necessary.

**5. Current Updates**

<details open>
<summary>Mockup</summary>
<br>

![Image](https://github.com/user-attachments/assets/62aa430b-22d2-4259-975d-efbb841d7b57)

</details>

**Comment by ExperimentsInHonesty** [comment]
I agree this will probably be fine unbolded.  Please add to deck

**Comment by thetanmancan** [comment]
This is the final design to be presented to stakeholders at today's meeting.
<details><summary>Final "Your project level is 0" infobox design</summary>

![Image](https://github.com/user-attachments/assets/d4db1595-02ac-4291-b976-7cc67fc7e680)
</details>

This design can be found in the "Final Design" section of [this Figma page](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18120-15251&t=TrlU0u2UcvalSyGh-1).

**Comment by thetanmancan** [comment]
@mjldesign, please update the [Style Guide Presentation](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit#slide=id.g2f74a24ec8f_0_8) and [Figma design system](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=3331-11987&t=TrlU0u2UcvalSyGh-1) with the changes from this story. I will close this issue once you've completed these tasks. Thank you.

**Comment by thetanmancan** [comment]
@mjldesign: Production wanted to discuss the Start Over button at today's meeting, so let's hold off on updating the presentation or design system until after then. Thanks.

**Comment by thetanmancan** [comment]
@mjldesign, I'm going to unassign you and move this issue to the ice box until we get an answer back from stakeholders from #2069 about how they want to go about potentially changing this page to a modal.

**Comment by marlenamellody** [comment]
**2025-02-04 stakeholder meeting: Suggestion was approved by stakeholders.** 

We recommend adding the ability to navigate to pg5 so they can “print summary”.

We could include these functionalities:

- Reset Project Text Button (without Clear Page)
- Navigate to Save Summary/Page 5

On the Save Summary page, we could have:
- Custom level 0 information
- A link to the ordinance
- Print Summary functionality
- Save Project functionality

**Comment by ExperimentsInHonesty** [comment]
bottom of the page will have

"<"  "3/5"  ">"

not
 "<" "Summary"

**Comment by marlenamellody** [comment]
![Image](https://github.com/user-attachments/assets/afae7793-505a-473b-b12f-19fa86808885)

**Comment by thetanmancan** [comment]
Hi @marlenamellody. Based on the mockup from [your most recent comment](https://github.com/hackforla/tdm-calculator/issues/1935#issuecomment-2641901677), it looks like this issue has been completed based on the recommendations from the [comment you made before that](https://github.com/hackforla/tdm-calculator/issues/1935#issuecomment-2638835207). Is this issue ready for design review?

**Comment by marlenamellody** [comment]
@thetanmancan, this is ready for design review. I will move it over to the visual review column.

**Comment by thetanmancan** [comment]
![Image](https://github.com/user-attachments/assets/5f49c689-ae5a-41a6-b319-ba877b313742)

**Comment by thetanmancan** [comment]
@marlenamellody Once you implement any relevant changes from this issue to the design system in the Figma and Google Slides presentation, you can close this issue. Thank you.
